src/frontends/pytorch/src/op_table.cpp

```diff
@@ -942,9 +949,12 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_fx() {
         {"aten.pixel_unshuffle.default", op::translate_pixel_unshuffle},
         {"aten.rand.default", op::translate_rand},
         {"aten.reciprocal.default", op::translate_reciprocal},
-        {"aten.reflection_pad1d.default", op::translate_reflection_pad_nd_fx},
-        {"aten.reflection_pad2d.default", op::translate_reflection_pad_nd_fx},
-        {"aten.reflection_pad3d.default", op::translate_reflection_pad_nd_fx},
+        {"aten.reflection_pad1d.default", op::translate_reflection_pad_nd},
+        {"aten.reflection_pad2d.default", op::translate_reflection_pad_nd},
+        {"aten.reflection_pad3d.default", op::translate_reflection_pad_nd},
+        {"aten.replicate_pad1d.default", op::translate_replicate_pad_nd},
+        {"aten.replicate_pad2d.default", op::translate_replicate_pad_nd},
+        {"aten.replicate_pad3d.default", op::translate_replicate_pad_nd},
```

Comment:
Tests for torch.export fail. I can see that names are wrong, but tests fail on torch side, please check them

Reply:
I am not sure but If I am trying to open the logs 
![image](https://github.com/user-attachments/assets/9ea65c3b-087c-4f01-916b-e769b1ea40a1)

its just loading & loading not showing the logs. How may I acces the failed logs.
Thank you

Reply:
I have corrected the names. Thank you